docs/deprecated.md

```diff
@@ -45,9 +43,9 @@ _hidden_, to discourage their use.
 
 ### Top-level network properties in NetworkSettings
 
-**Deprecated In Release: [v1.13.0](https://github.com/docker/docker/releases/tag/v1.13.0)**
+## Deprecated In Release: [v1.13.0](https://github.com/docker/docker/releases/tag/v1.13.0)
```

Comment:
One more change. Can you please make the link text here just be part of the title, and move the link down as a paragraph?

```
## Deprecated in Docker 1.13.0

[v1.13.0](https://github.com/docker/docker/releases/tag/v1.13.0)

...
```

This is because it doesn't make sense to try to have a link in a heading that will itself be a link. Sorry for the extra work, but it really will make this file more functional.

Reply:
Perhaps we should change the caption of that link to "v1.13.0 release notes" (or "Docker CE v1.13.0 release notes")

Reply:
I'd be OK with that too.

Reply:
Ok, I will make all these changes when I get home from work today.